types/recharts/index.d.ts

```diff
@@ -1,25 +1,38 @@
 // Type definitions for Recharts 0.22
 // Project: http://recharts.org/
 // Definitions by: Maarten Mulders <https://github.com/mthmulders/>
+//                 Raphael Mueller <https://github.com/rapmue/>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-import * as React from 'react';
+declare module 'recharts' {
```

Comment:
Then just remove this module wrapper from the file. Nothing else should need to be changed.